fleet_vehicle_configuration_event/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Add products "allowed" in vehicle configuration.
```

Comment:
IMO the module name is too generic for what it does.
This is a very specific requirement, so the module name should be specific too.

Moreover I have no idea how or why one would use this module 😅 
I think a better readme would help